app/src/main/java/com/infomaniak/mail/views/itemViews/UnreadItemView.kt

```diff
@@ -46,12 +46,14 @@ abstract class UnreadItemView @JvmOverloads constructor(
                 text = UiUtils.formatUnreadCount(unreadCount)
                 isVisible = !isPastilleDisplayed && unreadCount > 0
             }
+            binding.endIconLayout.isVisible = isPastilleDisplayed || unreadCount > 0
         }
 
     var isPastilleDisplayed = false
         set(isDisplayed) {
             field = isDisplayed
             setEndIcon(if (isDisplayed) pastille else null, R.string.contentDescriptionUnreadPastille)
+            binding.endIconLayout.isVisible = isPastilleDisplayed || unreadCount > 0
```

Comment:
Why did you need this two new visibility lines?

Reply:
You have good eyesight. Those lines were useful when I was forcing a size onto the chip container.